src/PlaywrightSharp.Tests/BrowserContextStorageStateTests.cs

```diff
@@ -64,9 +64,11 @@ public async Task ShouldCaptureLocalStorage()
         }
 
         [PlaywrightTest("browsercontext-storage-state.spec.ts", "should set local storage")]
-        [Fact(Timeout = TestConstants.DefaultTestTimeout)]
-        public async Task ShouldSetLocalStorage()
+        [Fact(Timeout = TestConstants.DefaultTestTimeout, Skip = "Needs to be implemented.")]
+        public void ShouldSetLocalStorage()
         {
+            // TODO: actually implement this
+            /*
```

Comment:
Do we have an issue to implement this?

Reply:
https://github.com/microsoft/playwright-sharp/issues/1310